src/helper/helper.js

```diff
@@ -55,6 +55,17 @@ const {type, hasOwn, isPlainObject} = goog.require('plain');
  */
 const DLH_DEBUG = goog.define('DLH_DEBUG', false);
 
+/**
+ * @typedef {!Object<string,{
+ *       listener: (function(!Object,!Object):undefined),
+ *       listenToPast: boolean,
+ *       processNow: boolean,
+ *       commandProcessors:
+ *         !Object<string,!Array<function(*):(!Object|undefined)>>
```

Comment:
I think this should be 
`!Object<string,!Array<function(...*):(!Object|undefined)>>`
If you just do `*` it just takes 1 parameter, while `...*` can take any number of parameters. Here and in some other places.

Reply:
Fixed!